controller/doc_manager.ml

```diff
@@ -172,16 +174,17 @@ end = struct
       (List.nth_opt pt_requests 0)
 
   (* Notification handling; reply is optional / asynchronous *)
-  let check ~ofn ~uri =
+  let check ~ofn ~concise ~uri =
     LIO.trace "process_queue" "resuming document checking";
     match Handle.find_opt ~uri with
     | Some handle ->
       let target = get_check_target handle.pt_requests in
       let doc = Fleche.Doc.check ~ofn ~target ~doc:handle.doc () in
       let requests = Handle.update_doc_info ~handle ~doc in
-      send_diags ~ofn ~doc;
-      (* Only if completed! *)
-      if completed ~doc then send_perf_data ~ofn ~doc;
+      if not concise then (
+        send_diags ~ofn ~doc;
+        (* Only if completed! *)
+        if completed ~doc then send_perf_data ~ofn ~doc);
```

Comment:
Maybe we could introduce finer grained options (and set them properly) as done for other configuration items?

I'd do that and remove the `concise` parameter, as we already have `Fleche.Config` for other options.

Reply:
Yeah that makes sense! Especially for warnings, it's not clear whether --concise should include them or not, having the user be able to specify a verbosity level would be good. I'll look into how Fleche.Config works.